src/style.css

```
@@ -1 +1,33 @@
-/* write your styles here */
+#header {
+  background-color: blue;
+}
+
+.important {
+  background-color: red;
+}
+
+.interesting {
+  background-color: green;
+}
+
+.surprising {
+  background-color: orange;
+}
+
+.footer {
+  background-color: black;
+}
+
+body {
+  margin: 0;
+}
+
+body > .block {
```

```suggestion
.block {
```
why you need body here? remove it